src/ts/utils/interface.ts

```diff
@@ -0,0 +1,21 @@
+interface productProps {
```

Comment:
```suggestion
interface ProductProps {
```
타입이나 인터페이스명은 일반적으로 [Pascal](https://stackoverflow.com/questions/29601960/typescript-use-pascalcasing-or-camelcasing-for-module-names)을 사용해요